SPECS/openssl/openssl.spec

```diff
@@ -242,16 +241,7 @@ mv %{buildroot}%{_sysconfdir}/pki/tls/misc/tsget %{buildroot}%{_bindir}
 
 # Rename man pages so that they don't conflict with other system man pages.
```

Comment:
Is this comment still valid with the current changes?

Reply:
Yes, this is still a valid comment. Now it refers to a smaller subset of man pages, since the removed step is already done by openssl's build system, but the gist is the same- we are renaming conflicts.